sites/example-project/tailwind.config.cjs

```diff
@@ -1,6 +1,22 @@
 const evidenceTailwind = require('@evidence-dev/tailwind').config;
 const { loadConfig } = require('@evidence-dev/plugin-connector/load-config');
 
+const fs = require('fs');
+const path = require('path');
+let presets = [evidenceTailwind];
+
+const altConfigFilenames = ['tailwind.config.js', 'tailwind.config.cjs', 'tailwind.config.mjs'];
+const altConfigFilepaths = altConfigFilenames.map((filename) => path.join('..', '..', filename));
+// Use find so that we can stop iteration
+altConfigFilepaths.find((file) => {
+	if (fs.statSync(file, { throwIfNoEntry: false })) {
+		presets.push(require(file));
```

Comment:
Would `require('tailwind.config.mjs')` work? might have to be a dynamic import

Reply:
Ah yeah this is a good shout, that will not work